packages/frontend-api/src/Resources/config/graphql-types/ProductDecorator.types.yml

```diff
@@ -0,0 +1,37 @@
+ProductDecorator:
+    type: object
+    decorator: true
+    config:
+        description: "Represents a category"
+        fields:
+            uuid:
+                type: "ID!"
+                description: "UUID"
+            name:
+                type: "String"
+                description: "Localized product name (domain dependent)"
+            shortDescription:
+                type: "String"
+                description: "Localized product short description (domain dependent)"
+            link:
+                type: "String!"
+                description: "Product link"
+            unit:
+                type: "String"
+                description: "Localized product unit name (domain dependent)"
+            availability:
```

Comment:
think this should be type

Reply:
Fixed https://github.com/shopsys/shopsys/pull/1471/commits/8354bdbf47855237480aeecffa5060bcbeee7ecc